cgi/session.pl

```diff
@@ -49,6 +49,7 @@
 if (defined $User_id) {
 
 	$template_data_ref->{user_name} =  $User{name};
+	$template_data_ref->{is_admin} = is_admin_user($User_id);
```

Comment:
In fact there's already an "admin" field that is set automatically for all templates:

in Display.pm:

```
sub process_template($$$) {

	my $template_filename = shift;
	my $template_data_ref = shift;
	my $result_content_ref = shift;

	# Add functions and values that are passed to all templates

[..]
	$template_data_ref->{admin} = $admin;
```

Reply:
https://github.com/openfoodfacts/openfoodfacts-server/blob/686f4fdf5d2b26209581c91050c7c7fc08845814/lib/ProductOpener/Display.pm#L380

Reply:
ok I removed this and use admin in template now.